### PR TITLE
Advanced form display fix.

### PR DESCRIPTION
**Pull request by sinukaarel**
Also...

Updated admin menu pages (deprecated functions).
Widget forms default language english if WPML not installed.
Pumped up version to 1.1.3

### CODE REVIEW

File: code/Template.php [hunk 1]
```diff
@@ -39,19 +39,23 @@ public function partial( $template, $params = array() ) {
...
 			return false;
 		}
-
+		
 		// Output template.
 		ob_start();
 		include( $file_name );
-		$output = ob_get_contents();
+		// Show advanced form when requested
+		if( isset($this->_vars['form']) && ! empty($this->_vars['form'])  && $showAdvanced){
```
Comment (kaittodesk) [inline code/Template.php]: This is not the right place for showing the advanced form. `Template` class is meant to be generic template renderer that does not have any other purpose than to render a template.

Comment (kaittodesk) [inline code/Template.php]: This is not the right place for showing the advanced form. `Template` class is meant to be generic template renderer that does not have any other purpose than to render a template.

Implement advanced form as it was before in commit https://github.com/sendsmaily/sendsmaily-wordpress-plugin/commit/6802f90104e7cd3109bbfdfe10a5a5816b42943a

File: html/form/advanced.phtml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <form id="smly" class="container" action="<?php echo SS_PLUGIN_URL; ?>/subscribe.php" method="post">
 	<p class="error" style="padding:15px;background-color:#f2dede;margin:0 0 10px;display:none"></p>
 	<p class="success" style="padding:15px;background-color:#dff0d8;margin:0 0 10px;display:none"><?php echo esc_html__( 'Thank you for subscribing to our newsletter.', 'wp_sendsmaily' ); ?></p>
-
+	<input type="hidden" name="lang" value="<?php echo ( defined( 'ICL_LANGUAGE_CODE' ) ) ? ICL_LANGUAGE_CODE : 'en'; ?>" />
```
Comment (kaittodesk) [inline html/form/advanced.phtml]: I think defaulting to English is not good. User's site might be in single language and the language may not be English. I would rather disable `lang` field entirely if `ICL_LANGUAGE_CODE` is not defined.

File: sendsmaily.php [hunk 3]
```diff
@@ -121,19 +121,24 @@ function smaily_subscribe_callback() {
 	$table_name = esc_sql( $wpdb->prefix . 'sendsmaily_config' );
 	$config = $wpdb->get_row( "SELECT * FROM `$table_name`" );
 
+	$form_values = [];
+	// Add custom form values to Api request if available
+	foreach($params as $key => $value){
```
Comment (kaittodesk) [inline sendsmaily.php]: Exclude fields provided by the system (i.e. keys provided by `$array`).

File: code/Template.php (hunk 1 above)
Comment (kaittodesk) [inline code/Template.php]: This is not the right place for advanced form rendering. `Template` class is meant to be generic template renderer.

Implement advanced form rendering as it was before commit https://github.com/sendsmaily/sendsmaily-wordpress-plugin/commit/6802f90104e7cd3109bbfdfe10a5a5816b42943a.

Comment (kaittodesk) [inline code/Template.php]: This is not the right place for advanced form rendering. `Template` class is meant to be generic template renderer.

Implement advanced form rendering as it was before commit https://github.com/sendsmaily/sendsmaily-wordpress-plugin/commit/6802f90104e7cd3109bbfdfe10a5a5816b42943a.

Comment (kaittodesk) [inline code/Template.php]: This is not the right place for rendering advanced form. `Template` class is meant to be generic template renderer.

Implement advanced form rendering as it was before commit https://github.com/sendsmaily/sendsmaily-wordpress-plugin/commit/6802f90104e7cd3109bbfdfe10a5a5816b42943a (advanced form existence was checked in the template).

File: html/form/advanced.phtml (hunk 2 above)
Comment (kaittodesk) [inline html/form/advanced.phtml]: Defaulting to English might not be the best approach. User's site can be in single language and that language might not be English.

I would exclude `lang` field entirely if `ICL_LANGUAGE_CODE` is not defined. Or to keep the previous behaviour and default `lang` to empty value.

File: sendsmaily.php [hunk 4]
```diff
@@ -121,19 +121,24 @@ function smaily_subscribe_callback() {
...
+		$form_values[$key] = $value;
```
Comment (kaittodesk) [inline sendsmaily.php]: Exclude fields provided by the system (i.e. keys from `$array` variable).

**Comment by kaittodesk** [comment]
Looks good.

Please review and improve code formatting where applicable according to Wordpress PHP Coding Standards (https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/).

**Comment by sinukaarel** [comment]
Fixes #5

**Pull request merged**